tests/testEMPsfApprox.py

```diff
@@ -0,0 +1,208 @@
+        exposure.setPsf(psf)
+        source = runMeasure(task, schema, exposure)
+        self.assertEqual(source.get(self.algName + "_flag"), True)
+        self.assertEqual(source.get(self.algName + "_flag_rangeError"), False)
+        self.assertEqual(source.get(self.algName + "_flag_maxIters"), True)
+        self.assertEqual(source.get(self.algName + "_flag_noPsf"), False)
+
+    #   Test to be sure that we can catch the missing psf error
+    def testMissingPsf(self):
+        msConfig = self.makeConfig()
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        task = measBase.SingleFrameMeasurementTask(schema=schema, config=msConfig)
+        exposure = afwImage.ExposureF(os.path.join(self.dataDir, "exp.fits"))
+        #   Strip the psf
+        exposure.setPsf(None)
+        source = runMeasure(task, schema, exposure)
+        self.assertEqual(source.get(self.algName + "_flag"), True)
+        self.assertEqual(source.get(self.algName + "_flag_rangeError"), False)
+        self.assertEqual(source.get(self.algName + "_flag_maxIters"), False)
+        self.assertEqual(source.get(self.algName + "_flag_noPsf"), True)
+
+    #   Test that the EmPsfApprox plugin produces a more or less reasonable result
+    def testEMPlugin(self):
+        msConfig = self.makeConfig()
+        msConfig.plugins[self.algName].nGauss = 2
+        schema = afwTable.SourceTable.makeMinimalSchema()
+        task = measBase.SingleFrameMeasurementTask(schema=schema, config=msConfig)
+        exposure = afwImage.ExposureF(os.path.join(self.dataDir, "exp.fits"))
+
+        #   this is a double gaussian with a .7/.3 ratio of inner to outer
+        #   we expect ixx = iyy = sigma*sigma
+        psf = makePsf(67, 4.0, .7, sigma2=10.0, mult2=.3)
+        exposure.setPsf(psf)
+        source = runMeasure(task, schema, exposure)
+
+        #   Be sure there were no failures
+        self.assertEqual(source.get(self.algName + "_flag"), False)
+        self.assertEqual(source.get(self.algName + "_flag_rangeError"), False)
+        self.assertEqual(source.get(self.algName + "_flag_maxIters"), False)
+        self.assertEqual(source.get(self.algName + "_flag_noPsf"), False)
+
+        self.msfKey = lsst.shapelet.MultiShapeletFunctionKey(schema[self.algName],
+                                                             lsst.shapelet.HERMITE)
+
+        #   check the two component result to be sure it is close to the input PSF
+        #   we don't control the order of EmPsfApprox, so order by size.
+        msf = source.get(self.msfKey)
+        components = msf.getComponents()
+        self.assertEqual(len(components), 2)
+        comp0 = components[0]  
+        comp1 = components[1]
+        flux0 = comp0.getCoefficients()[0]  
+        flux1 = comp1.getCoefficients()[0]  
+        if flux0 < flux1:
+            temp = comp1
+            comp1 = comp0
+            comp0 = temp
+        self.assertClose(flux0/flux1, 7.0/3.0, rtol = .05)
```

Comment:
Can you write a comment or something and justify the tolerances on these number? In the future they may change for some reason, or someone my need to update the test and it would be useful to have some context on what these numbers mean and what the tolerances are.